fenix/app/src/main/java/org/mozilla/fenix/library/history/HistoryFragment.kt

```diff
@@ -317,10 +317,7 @@ class HistoryFragment : LibraryPageFragment<History>(), UserInteractionHandler,
                 (selectedItem as? History.Regular)?.url ?: (selectedItem as? History.Metadata)?.url
             }
 
-            (activity as HomeActivity).apply {
-                browsingModeManager.mode = BrowsingMode.Private
-                supportActionBar?.hide()
-            }
+            (activity as HomeActivity).supportActionBar?.hide()
```

Comment:
Note that `openItemsInNewTab` which is called on the line before this dispatches the PBM state update, so I removed the redundancy here.

Reply:
Now, this opens a new tab which includes the browsing mode and triggers the `BrowserStoreBinding`